include/hse/hse.h

```diff
@@ -941,9 +944,9 @@ hse_kvdb_txn_state_get(struct hse_kvdb *kvdb, struct hse_kvdb_txn *txn);
  * of keys in the KVS whose first @p filter_len bytes match the @p filter_len
  * bytes pointed to by @p filter.
  *
- * A prefix cursor is created when:
- * @li KVS was created with @p pfx_len > 0 (i.e., it is a prefix KVS), and
- * @li @p filter != NULL and @p filter_len >= @p pfx_len.
+ * A prefix cursor is created when the following two conditions are met:
+ * @li KVS was created with a prefix length > 0 (i.e., it is a prefix KVS)
+ * @li @p filter != NULL and @p filter_len >= @p prefix.length.
  *
  * Otherwise, a non-prefix cursor is created.
```

Comment:
This terminology does not align with what we agreed on for the docs.  There are "non-transaction cursors" and "transaction cursors", either of which can take an optional (prefix) filter.  There is no "prefix cursor".

See https://hse-project.github.io/3.x/dev/concepts/#cursors.

I noted later in this file where we use the correct terminology.

Reply:
Thanks. I will update this.